README.md

```diff
@@ -1,5 +1,9 @@
 # HR-TIL
 
+![til](https://raw.githubusercontent.com/michaeldimmitt/hr-til/readme-oauth-instructions/app/assets/images/banner.png)
+
+<!--![til](https://raw.githubusercontent.com/hashrocket/hr-til/master/app/assets/images/banner.png)-->
```

Comment:
I am just going to go ahead and convert it over to Hashrocket image asset. Since you said it looked good. My understanding is that it will then only show up after the commit is merged.